aquacomputer_d5next.c

```diff
@@ -480,18 +483,6 @@ static const char *const label_aquaero_aquabus_temp_sensors[] = {
 	"Aquabus sensor 20"
 };
 
-/* Labels for Octo and Quadro (except speed) */
-static const char *const label_fan_speed[] = {
```

Comment:
This is in fact not used by Quadro any more (there's a separate array further down in the file)